src/Commands/TerminusCommand.php

```diff
@@ -21,21 +24,34 @@
      * @var boolean True if the command requires the user to be logged in
      */
     protected $authorized = false;
+    /**
+     * @var LoggerInterface
+     */
+    private $logger;
 
     /**
      * TerminusCommand constructor
-     *
-     * @param Config $config Terminus configuration object
      */
-    public function __construct($config)
+    public function __construct()
     {
-        $this->config = $config;
+        $this->config = new Config();
+        $this->setLog(new ConsoleLogger($this->output()));
```

Comment:
Also, this line (`$this->setLog(new ConsoleLogger($this->output()));`) is the source of the verbosity bug.  What's happening here is that Robo injects the output object via the technique above ^^; it uses setOutput() in OutputAwareTrait, since you implement OutputAwareInterface.  The result of this is that $this->output() is not available in your constructor, as the constructor is called to set up the object before its setOutput() method is called.

n.b. if you construct your own objects, you'll need to inject the input and output objects yourself.